circle.yml

```diff
@@ -0,0 +1,23 @@
+machine:
+  environment:
+    YARN_VERSION: 0.18.1
+    PATH: "${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+
+dependencies:
+  pre:
```

Comment:
Please use the simplified yarn setup that's now available: https://circleci.com/docs/yarn/#full-example